easybuild/easyblocks/generic/rpm.py

```diff
@@ -91,58 +128,29 @@ def configure_step(self):
             ver = res.groupdict()['version']
 
             if LooseVersion(ver) >= LooseVersion('4.8.0'):
```

Comment:
actually, I only know this is needed starting with the version of `rpm` that was on SL6 at the time